File: MachineProblems/Code/MP7.java
import java.util.Scanner;

public class Mp2 {
    public static void main(String[] args) {
        int even = 0;
        int odd = 0;
        int pos = 0;
        int neg = 0;

        Scanner input = new Scanner(System.in);

        int[] num = new int[15];


        for (int i = 0; i < num.length; i++) {
            num[i] = input.nextInt();
        }

        for (int i = 0; i < num.length; i++){
            if (num[i] % 2 == 0) {
                even++;
            } else {
                odd++;
            }

            if (num[i] > 0){
                pos++;
            } else if (num[i] < 0) {
                neg++;
            }
        }

        System.err.println();
        System.err.println("Count of Even Numbers: "+even);
        System.err.println("Count of Odd Numbers: "+odd);
        System.err.println("Count of Positive Numbers: "+pos);
        System.err.println("Count of Negative Numbers: "+neg);
    }
}

File: MachineProblems/Code/MP7.py

numbers = []
numOfEvenNumbers =  0
numOfOddNumbers = 0
numOfPositiveNumbers = 0
numOfNegativeNumbers = 0

for i in range(15):
    number = int(input())

    if number % 2 == 0:
        numOfEvenNumbers+=1
    else:
        numOfOddNumbers+=1

    if number > 0:
        numOfPositiveNumbers+=1
    else:
        numOfNegativeNumbers+=1

print(f"Count of Even Numbers: {numOfEvenNumbers}")
print(f"Count of Odd Numbers: {numOfOddNumbers}")
print(f"Count of Positive Numbers: {numOfPositiveNumbers}")
print(f"Count of Negative Numbers: {numOfNegativeNumbers}")

    